### ISSUE TITLE
Cannot load twemoji color font

### ISSUE THREAD

**Issue by misuke0** [ocornut/imgui#6607]
**Version/Branch of Dear ImGui:**

Version: 1.89.7
Branch: master

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_dx11.cpp + imgui_impl_win32.cpp
Compiler: Visual Studio 2022
Operating System: Windows

**My Issue/Question:**

I cannot load the twemoji font (https://github.com/twitter/twemoji), I followed the instructions here: https://github.com/ocornut/imgui/blob/master/docs/FONTS.md#using-colorful-glyphsemojis. With seguiemj.ttf it works but not with twemoji.

```cpp
ImGuiIO& io = ImGui::GetIO();
auto font = io.Fonts->AddFontFromFileTTF("c:\\Windows\\Fonts\\segoeui.ttf", 18);

static ImWchar ranges[] = { 0x1, 0x1FFFF, 0 };
static ImFontConfig cfg;
cfg.OversampleH = cfg.OversampleV = 1;
cfg.MergeMode = true;
cfg.FontBuilderFlags |= ImGuiFreeTypeBuilderFlags_LoadColor;
io.Fonts->AddFontFromFileTTF("c:\\users\\misuke\\downloads\\TwitterColorEmoji-SVGinOT.ttf", 18f, &cfg, ranges);
```
It throws an exception in imgui_freetype.cpp (https://github.com/ocornut/imgui/blob/master/misc/freetype/imgui_freetype.cpp#L247):
```cpp
IM_ASSERT(slot->format == FT_GLYPH_FORMAT_OUTLINE || slot->format == FT_GLYPH_FORMAT_BITMAP);
```
If I remove this assertion, the font atlas contains only a few icons from twemoji
**Screenshots/Video**

![image](https://github.com/ocornut/imgui/assets/139678297/2edd2915-a1e4-4b0f-80ac-66c9522dc219)

**Comment by PathogenDavid** [comment]
Twemoji is an SVG font, which aren't supported by FreeType out of the box. (Hence the failed assert.)

There's a PR to enable support via LunaSVG, see https://github.com/ocornut/imgui/pull/6591

**Comment by misuke0** [comment]
@PathogenDavid thanks for the quick response, that works for me

**Comment by ocornut** [comment]
Merged Lunasvg support in #6591
